core-services/prow/02_config/_plugins.yaml

```diff
@@ -1,4 +1,6 @@
 blunderbuss:
+  ignore_authors:
```

Comment:
It's a controversial change to ignore for all the orgs all the repos.

Reply:
Fair, but I get an error if I put the Blunderbuss configuration anywhere else:

>time="2022-08-27T12:00:19Z" level=fatal msg="could not update Prow plugin configuration" error="could not load Prow plugin configuration: failed to merge config from /config/stolostron/_pluginconfig.yaml into main config: supplemental plugin configuration has config that doesn't support merging:   &plugins.Configuration{\n  \t... // 3 identical fields\n  \tApprove:   {{Repos: {\"stolostron\"}, RequireSelfApproval: &false, LgtmActsAsApprove: true, CommandHelpLink: \"htps://go.k8s.io/bot-commands\"}},\n  \tBlockades: nil,\n  \tBlunderbuss: plugins.Blunderbuss{\n  \t\t... // 2 identical fields\n  \t\tExcludeApprovers:      false,\n  \t\tUseStatusAvailability: false,\n- \t\tIgnoreDrafts:          true,\n+ \t\tIgnoreDrafts:          false,\n- \t\tIgnoreAuthors:         []string{\"magic-mirror-bot\"},\n+ \t\tIgnoreAuthors:         nil,\n  \t},\n  \tBugzilla:      {},\n  \tBranchCleaner: {},\n  \t... // 23 identical fields\n  }\n"
make: *** [prow-config] Error 1

Is there somewhere I can update the sharding logic to make this work?

Reply:
You can't, unfortunately, it is not implemented. Does your bot account (GH App?) is getting assigned to PRs?

Reply:
Yes, our bot opens PRs so we don't want to be assigned as reviewers to its PRs.

Reply:
@jmguzik Is there a path forward here?

I understand it's controversial, but to my knowledge usernames are also unique and it seems it can't be too disruptive to not assign reviewers to a username, especially one of an app under our management.

Alternatively, I'm happy to see about submitting a PR to make the sharding logic work for blunderbuss if you can point me to the sharding code.

Reply:
Generally, we do not allow changes in our main prow config that are targeting specific teams. We do not want to create a chaotic situation and make precedences.
We had users really interested in creating shards for some additional functions, beyond what is shared currently, they even implemented these things in https://github.com/kubernetes/test-infra/tree/master/prow/plugins I would say we should do it this way. If you are really interested and it is currently, unfortunately, impossible, you can implement it.

Reply:
Understood--thanks for the explanation and the link! I'll take a look to see about implementing it.